site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -56,6 +58,17 @@ const Planner: FC = () => {
       planners: collapseAllPlanners(allPlanData),
       transfers: transfers,
     };
+    const coursebagStrings = coursebag.map((course) => course.id);
+    // if logged in
+    if (cookies.user !== undefined) {
+      // save data to account
+      const mongoRoadmap: MongoRoadmap = {
+        _id: cookies.user.id,
+        roadmap: roadmap,
+        coursebag: coursebagStrings,
+      };
+      axios.post('/api/roadmap', mongoRoadmap);
+    }
```

Comment:
duplicate from merging

```suggestion
    const coursebagStrings = coursebag.map((course) => course.id);
    // if logged in
    if (cookies.user !== undefined) {
      // save data to account
      const mongoRoadmap: MongoRoadmap = {
        _id: cookies.user.id,
        roadmap: roadmap,
        coursebag: coursebagStrings,
      };
      axios.post('/api/roadmap', mongoRoadmap);
    }
```